docs/ConfigurationFile.md

```diff
@@ -119,6 +119,23 @@ exports.config = {
     // Level of logging verbosity: trace | debug | info | warn | error
     logLevel: 'info',
     //
+    // Set specific log levels per logger
+    // use 'silent' level to disable logger
+    logLevels: {
+        webdriver: 'info',
+        'wdio-applitools-service': 'info'
+    },
+    //
+    // If outputDir is provided WebdriverIO can capture driver session logs
+    // it is possible to configure which logTypes to include/exclude.
+    driverLog: {
+        // include: ['logcat'], // only include certain types of logs
+        exclude: ['bugreport', 'server']
+    },
+    //
+    // Set directory to store all logs into
+    outputDir: __dirname,
+    //
```

Comment:
Can we add this (completely commented out though) to https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cli/src/templates/wdio.conf.tpl.ejs ?